rakelib/artifacts.rake

```diff
@@ -378,6 +378,7 @@ namespace "artifact" do
 
   task "generate_build_metadata" do
     require 'time'
+    require 'tempfile'
```

Comment:
review note: when firing this rake task alone, we get uninitialized tempfile error.